### PR TITLE
ffmpeg: split ffplay

### PR DESCRIPTION
**Pull request by paper42**
ffplay depends on SDL2 which has a lot of dependencies, some systems
like servers may wish to install ffmpeg, but without ffplay to avoid
the SDL2 dependency.

I think it would be nice if ffmpeg didn't depend on ffplay, but that could break for some people. Should I remove the dependency?

#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/ffmpeg/template [hunk 1]
```diff
@@ -24,6 +24,7 @@ makedepends="zlib-devel bzip2-devel freetype-devel alsa-lib-devel libXfixes-deve
  $(vopt_if nvenc nv-codec-headers) $(vopt_if sndio sndio-devel)
  $(vopt_if dav1d libdav1d-devel) $(vopt_if zimg zimg-devel)
  $(vopt_if webp libwebp-devel) $(vopt_if sofa libmysofa-devel)"
+depends="ffplay"
```
Comment (ericonr) [inline srcpkgs/ffmpeg/template:27]: So make it possible to `ignorepkg` it, basically? And I think this should use `ffplay>=${version}_${revision}`

**Review by ericonr — COMMENTED** [review]
@jnbr

File: srcpkgs/ffmpeg/template [hunk 2]
```diff
@@ -183,3 +184,10 @@ ffmpeg-devel_package() {
 		vmove usr/share/man/man3
 	}
 }
+ffplay_package() {
+	short_desc+="FFmpeg video resampling library"
```
Comment (ericonr) [inline srcpkgs/ffmpeg/template:188]: shortdesc seems wrong, this isn't the library...

Comment (paper42) [reply srcpkgs/ffmpeg/template]: oops, good catch

**Pull request merged**